content/zh/docs/_index.md

```diff
@@ -4,7 +4,7 @@ menu: { main: { weight: 10 } }
 default_lang_commit: 6e35a949
 ---
 
-OpenTelemetry 也被称为 OTel，是一个供应商中立的、开源的[可观测性](concepts/observability-primer/#what-is-observability)框架，
+OpenTelemetry 也被称为 OTel，是一个供应商中立的、开源的[可观测性](concepts/observability-primer)框架，
```

Comment:
is there a reason why the anchor has been removed? Is it possible to use an anchor with Chinese symbols? If not you can annotate the title with an anchor name:

```
## Title of the section {#what-is-observability}
```

Reply:
Thanks for catching that @svrnm. Right, anchors _must not be removed_. URLs should match what is in the English pages. 

@open-telemetry/docs-zh-approvers - please ensure that you follow the best practices outlined in the localization documentation, in this case in particular, see https://opentelemetry.io/docs/contributing/localization/#heading-anchors.

Reply:
Thanks for pointing this out, I have updated line9 to include the ` {#what-is-observability}`, could you have a look if it's good now?

Reply:
Yes, looks good now, thank you.